From 2362853794de52719202b748d58aad781f3e1476 Mon Sep 17 00:00:00 2001
From: Roshan Pius <rpius@google.com>
Date: Thu, 18 Jan 2018 14:24:13 -0800
Subject: [PATCH 5/9] klte-common: Don't start supplicant with interfaces

Framework will now add interface to supplicant (via HIDL) when it
needs to control an interface, so don't specify them in the startup
params.

Bug: 69426063
Test: Device boots up and able to connect to wifi.
Change-Id: I817d28093f8b982e3806c212babd888c0f5eb7c0
Signed-off-by: Kevin F. Haggerty <haggertk@lineageos.org>
---
 rootdir/etc/init.qcom.rc | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/rootdir/etc/init.qcom.rc b/rootdir/etc/init.qcom.rc
index 136ea08..7250889 100644
--- a/rootdir/etc/init.qcom.rc
+++ b/rootdir/etc/init.qcom.rc
@@ -917,10 +917,6 @@ service qmuxd /vendor/bin/qmuxd
     group radio audio bluetooth gps oem_2950 log
 
 service wpa_supplicant /vendor/bin/hw/wpa_supplicant \
-    -iwlan0 -Dnl80211 -c/data/misc/wifi/wpa_supplicant.conf \
-    -I/vendor/etc/wifi/wpa_supplicant_overlay.conf -N \
-    -ip2p0 -Dnl80211 -c/data/misc/wifi/p2p_supplicant.conf \
-    -I/vendor/etc/wifi/p2p_supplicant_overlay.conf \
     -O/data/vendor/wifi/wpa/sockets -puse_p2p_group_interface=1 \
     -g@android:wpa_wlan0
     class main
-- 
2.17.1

